lib/bankaccount.rb

```diff
@@ -0,0 +1,9 @@
+module BankAccount
+  require 'pathname'
+
+  ROOT_DIR = Pathname.new(__dir__).parent
+  require ROOT_DIR.join('lib/bankaccount/account')
+  require ROOT_DIR.join('lib/bankaccount/savingsaccount')
+  require ROOT_DIR.join('lib/bankaccount/checkingaccount')
+
+end
```

Comment:
Using Pathname is pretty heavyweight. Since rspec already does a lot of the proper file loading for you, you should just be able to `require 'bankaccount/account` etc.